doc/api/errors.markdown

```diff
@@ -6,13 +6,13 @@ Applications running in Node.js will generally experience four categories of
 errors:
 
 - Standard JavaScript errors such as:
-  - [`EvalError`][]: thrown when a call to `eval()` fails.
-  - [`SyntaxError`][]: thrown in response to improper JavaScript language
+  - {EvalError} : thrown when a call to `eval()` fails.
```

Comment:
Forgot to mention #5325. This is why actually I created PR. #5325 in turn was caused by #5322